python/example/single_cell_bpo/l5pc/Readme.md

```diff
@@ -0,0 +1,13 @@
+# BluePyOpt layer-5 pyramidal cell with axon-replacement
+
+This example was created in an environment with the `bluepyopt` package installed (see [installation instructions](https://github.com/BlueBrain/BluePyOpt#installation)). A cell model with parameters as published by [Markram et al., "Reconstruction and simulation of neocortical microcircuitry", Cell 163.2 (2015): 456–492](http://www.cell.com/abstract/S0092-8674%2815%2901191-5) (see [L5PC.ipynb](https://github.com/BlueBrain/BluePyOpt/blob/master/examples/l5pc/L5PC.ipynb)) can be exported with
```

Comment:
One last question on this: the licensing is unproblematic I hope?

Reply:
I'd like to forward this to @wvangeit - is it ok to add this exported L5PC model as an example as described [here](https://github.com/lukasgd/arbor/tree/bluepyopt_integration/python/example/single_cell_bpo/l5pc) to the Arbor repo?

Reply:
I assume this falls under the BSD license of the entire repo? For me it's ok if you keep the reference to the original notebooks.
Are you storing any of the MOD files of the L5PC, remember that these have their own licenses.

Reply:
Thank you. I didn't store any of the L5PC mechs in Arbor, but rather took the ones from the [built-in BBP catalogue](https://github.com/arbor-sim/arbor/tree/master/mechanisms/bbp). This is also the ones that I used in the Arbor-Neuron validation (and actually I have even tested the `K_Tst` and `K_Pst` with Neuron to make sure the difference between the simulators we found is not due to the NMODL code).